sort_test.go

```diff
@@ -88,6 +120,13 @@ func TestExternalSortSuccess(t *testing.T) {
 			if err != nil {
 				t.Fatal(err.Error())
 			}
+
+			data := bytesToLine(output)
+
+			if !reflect.DeepEqual(data, tc.expectedData) {
```

Comment:
You can instead use testutils.Compare()

Reply:
testutils is not a dependency of this project (and is a private repo)